### DISCUSSION TITLE
Can I show captions in the sidebar for child sections?

### DISCUSSION THREAD

**Discussion by stonecharioteer** [pradyunsg/furo#550] [Questions] [answered]
I already use the captions in the sidebar at the root level of [my website](https://stonecharioteer.com] to separate the sections. However, I have child pages with toctrees that also have captions and was wondering if it would be possible to render those captions as well in the sidebar?

**Comment by stonecharioteer** [comment]
![furo](https://user-images.githubusercontent.com/11478411/194821402-89b4cf9d-96ac-435d-aedc-b6cdc9c60529.png)
Here's an image of what I have right now (this isn't live). I was hoping I could show `talks` and `proposals` in the sidebar as a separator similar to how the main navbar renders.

**Comment by pradyunsg** [comment, marked as answer]
Not as of today, no.

https://github.com/sphinx-doc/sphinx/issues/10697

**Comment by stonecharioteer** [comment]
Ah, thanks!
